frontend/src/App.css

```
@@ -2651,16 +2689,158 @@ code {
+}
+
+.kanban-card-section-content strong {
+  font-weight: 600;
+}
+
+.kanban-card-section-content code {
+  font-size: 11px;
+  background: var(--color-fill-tertiary);
+  padding: 1px 4px;
+  border-radius: 3px;
+  font-family: 'SF Mono', 'SFMono-Regular', 'Fira Code', monospace;
```

Stylelint (17.11.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)